credoai/utils/credo_api_utils.py

```diff
@@ -7,25 +7,23 @@
 from dotenv import dotenv_values
 from json_api_doc import deserialize, serialize
 
+CREDO_URL = "https://api.credo-qa.com"
+
 def read_config():
     config_file = os.path.join(os.path.expanduser('~'), 
                                '.credoconfig')
     if not os.path.exists(config_file):
         # return example
-        config = {
-            'TENANT': 'empty',
-            'CREDO_URL': 'empty',
-            'API_KEY': 'empty'
-        }
+        config = {'API_KEY': 'empty'}
     else:
         config = dotenv_values(config_file)
-    config['API_URL'] = os.path.join(config['CREDO_URL'], "api/v1/credoai")
+    config['API_URL'] = os.path.join(CREDO_URL, "api/v1/credoai")
     return config
 
 def exchange_token():
-    data = {"api_token": CONFIG['API_KEY'], "tenant": CONFIG['TENANT']}
+    data = {"api_token": CONFIG['API_KEY'], "tenant": 'credoai'}
```

Comment:
The tenant is only credoai for Credo employees. @emoralesb05 - you mentioned that it is not actually necessary to provide this?

Reply:
I could not get an access token without it. Something is weird

Reply:
The UI gets the tenant from the JWT claims calling `/token` to get the JWT. I pinged @aproxacs on the other comments to see if we are missing something.